### PR TITLE
Vagrant support for provisioning one module

### PR DESCRIPTION
**Pull request by ingwarsw**
- "vagrant provision" should be able to update all hermes modules
- "vagrant provision --provision-with MODULE" should update only that module
- vagrant provision dont brake when doing it multiple times

### CODE REVIEW

**Comment by dankraw** [comment]
great! :+1:

File: vagrant_provisioning/install_hermes_module.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+/vagrant/gradlew -q -p /vagrant/hermes-${module} distZip -Pdistribution
+
+echo -n "Stopping module [$module]..."
+supervisorctl stop hermes-${module} || true
+
+echo "Installing module [$module]..."
+
+rm -rf /opt/hermes-${module}
+unzip -q /vagrant/hermes-${module}/build/distributions/hermes-${module}-*.zip -d /opt/
```
Comment (chemicL) [inline vagrant_provisioning/install_hermes_module.sh]: -o might help

File: vagrant_provisioning/start.sh [hunk 2]
```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env bash
 
-supervisord -c /etc/supervisord.conf -u root
+if pgrep supervisord >/dev/null 2>&1; then
+    echo "Supervisord is already running..."
+else
+    echo "Running supervisord..."
```
Comment (chemicL) [inline vagrant_provisioning/start.sh]: "Starting supervisord" would be more meaningful.

**Comment by chemicL** [comment]
Just minor remarks, but this is very helpful, thanks :-) :+1:

**Comment by ingwarsw** [comment]
Now hermes-console is also stopping correctly.

**Comment by adamdubiel** [comment]
:+1:

**Comment by adamdubiel** [comment]
resolve the conflicts before merge

**Comment by ingwarsw** [comment]
Done.. Was no conflicts.. but github saw something strange...

**Pull request merged**